### PR TITLE
Add tests for console domain.

### PR DESCRIPTION
**Pull request by DrMarcII**
Run tests on travis.

### CODE REVIEW

**Comment by DrMarcII** [comment]
This isn't ready yet. Not having much luck getting chrome to run in the background on travis for some reason.

**Comment by DrMarcII** [comment]
This is ready for review and merging.

File: tool/travis.sh [hunk 1]
```diff
@@ -10,3 +10,14 @@ set -e
...
+/usr/bin/chromium-browser --no-sandbox --remote-debugging-port=9222 \
+    --disable-background-networking --disable-client-side-phishing-detection \
+    --disable-component-update --disable-default-apps --disable-hang-monitor \
+    --disable-prompt-on-repost --disable-sync --disable-web-resources --enable-logging \
+    --ignore-certificate-errors --log-level=0 --metrics-recording-only --no-first-run \
+    --password-store=basic --safebrowsing-disable-auto-update \
+    --safebrowsing-disable-download-protection --test-type=browser --use-mock-keychain &
+
+pub global activate test
+pub global run test
```
Comment (DrMarcII) [inline tool/travis.sh]: done.

**Comment by devoncarew** [comment]
lgtm! It's great to have tests for this project now.

**Pull request merged**